scripts/lib/emoji_utils.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+from __future__ import division, print_function
+import hashlib
+import os
+import shutil
+import sys
+
+import xml.etree.ElementTree as ET
+from six import unichr
+from PIL import Image, ImageDraw, ImageFont
+
+if False:
+    from typing import Dict, Union, Text
+
+AA_SCALE = 8
+SIZE = (136, 136)
+BIG_SIZE = tuple([x * AA_SCALE for x in SIZE])
+
+
+class MissingGlyphError(Exception):
+    pass
+
+def color_font(name, code_point, code_point_to_fname_map):
```

Comment:
ohk, actually I thought they might be actually promptly used so I did the refactoring. If you think they shouldn't be moved I am happy to move just `get_success_stamp()` to the lib file.